tests/unit/model-data-test.js

```diff
@@ -184,14 +185,26 @@ module('unit/model-data', function(hooks) {
       };
       this.grandchild1ModelData = new M3ModelData(
         'com.exmaple.bookstore.book',
-        'grandchild1_1',
+        'child1_1',
```

Comment:
what's going on here?  now there's a mismatch between the example id and the variable name

Reply:
Adjust the variable names to reflect the `id`.